src/helpers/eventBridge.js

```diff
@@ -87,6 +92,8 @@ export default class EventBridge {
       })
       .promise();
 
+    await this.getEvents(); // need to clear this manual published event from the SQS observer queue.
```

Comment:
Do we always want to delete the events? 

It could be worth splitting this into two functions (eg. peekEvents, popEvents).
We would have to set the VisibilityTimeout to 0 when we create the queue so that peek doesn't change the visibly